l10n/DEFAULT/CavCaller.lua

```diff
@@ -0,0 +1,11 @@
+-- Function to call in the various functions defined in CavUtils, without having to call the entirerity of CavUtils.
+
+-- Disable player settings via MOOSE
+_SETTINGS:SetPlayerMenuOff()
+
+-- Admin Menu Tester, REMOVE BEFORE PUSH TO SERVER
+--local DemoMenu = CavUtils.AddAdminforAll()
+-- Adds in the admin menu via MOOSE function
+-- Sets the admin1 variable to the group "Admin-1"
+local admin1 = GROUP:FindByName('Admin-1')
+local MenuAdd = CavUtils.AddAdminMenu(admin1)
```

Comment:
From testing, I'm not seeing this option menu in the admin slot. Is this suppose to be working in this PR? Also line 6, were you suppose to remove something?

Reply:
We should just disable this functionality until we can get the admin menu working.

Reply:
No not working in current version. Need to troubleshoot more. Line 6 was commented out or shouldve been. Was just a tester function to add to the coalition, but its a bug i need to fix.

Reply:
Shouldn't we comment out line 11 too then?

Reply:
Ive left line 11 in, will continue to modify. We can comment it out but it looks harmless for now. I think its something wrong with my AddAdminMenu function in CavUtils. Need to look more into the moose scripting info.